### ISSUE TITLE
Issue: ByBit and Bitmart fetch_ohlcv prioritizes 24 hours turnover rate over volume

### ISSUE THREAD

**Issue by viper7882** [ccxt/ccxt#10768]
Hi @kroitor,
 
It seems like ByBit fetch_ohlcv is prioritizing 24 hours volume turn over rate over the actual volume. It has caused the volume returned by OHLCV exploded.

We saw other exchanges are prioritizing volume over 24 hours turnover rate. We hope you could get someone to look into this issue. Thank you in advance.

**High Level Overview**
Suspected problematic codes:
1. https://github.com/ccxt/ccxt/blob/master/python/ccxt/bybit.py#L784
```python
self.safe_number_2(ohlcv, 'turnover', 'volume'),
```

2. https://github.com/ccxt/ccxt/blob/master/python/ccxt/bitmart.py#L701
```python
baseVolume = self.safe_number_2(ticker, 'base_volume_24h', 'base_coin_volume')
quoteVolume = self.safe_number_2(ticker, 'quote_volume_24h', 'quote_coin_volume')
```

Expected code for 1,
```python
self.safe_number_2(ohlcv, 'volume', 'turnover'),
```

Expected code for 2,
```python
baseVolume = self.safe_number_2(ticker, 'base_coin_volume', 'base_volume_24h')
quoteVolume = self.safe_number_2(ticker, 'quote_coin_volume', 'quote_volume_24h')
```

**Reference codes**
1. https://github.com/ccxt/ccxt/blob/master/python/ccxt/bibox.py#L282
```python
baseVolume = self.safe_number_2(ticker, 'vol', 'vol24H')
```

2. https://github.com/ccxt/ccxt/blob/master/python/ccxt/bitget.py#L1091
```python
baseVolume = self.safe_number_2(ticker, 'amount', 'volume_24h')
```

3. https://github.com/ccxt/ccxt/blob/master/python/ccxt/gateio.py#L1376
```python
baseVolume = self.safe_number_2(ticker, 'base_volume', 'volume_24h_base')
quoteVolume = self.safe_number_2(ticker, 'quote_volume', 'volume_24h_quote')
```

4. https://github.com/ccxt/ccxt/blob/master/python/ccxt/mexc.py#L764
```python
baseVolume = self.safe_number_2(ticker, 'volume', 'volume24')
```

**Details**
- OS: Windows 11
- Programming Language version: Python v3.9
- CCXT version: 1.62.87

```python
import inspect
import time
from pprint import pprint

import ccxt

symbol_name = "BTC/USDT"

exchange = ccxt.bybit({
    'apiKey': api_key,
    'secret': api_secret,
    'nonce': lambda: str(int(time.time() * 1000)),
    'enableRateLimit': True,
})

# since = int(time.time()) - 60 * 1000
since = 1638511140 * 1000

result = exchange.fetch_ohlcv(
    symbol=symbol_name, timeframe="1m", since=since, limit=2
)

print("{} Line: {}: result:".format(
    inspect.getframeinfo(inspect.currentframe()).function,
    inspect.getframeinfo(inspect.currentframe()).lineno))
pprint(result)
```

```
D:\xxx\Issue\ccxt\venv\Scripts\python.exe D:/xxx/Issue/ccxt/get_klines.py
<module> Line: 71: result:
[[1638511140000, 56741.0, 56756.5, 56731.0, 56741.0, 2094167.757],
 [1638511200000, 56741.0, 56824.0, 56741.0, 56823.0, 828316.443]]

Process finished with exit code 0
```

In comparison with ByBit's pybit over the identical time:
```
D:\xxx\Issue\PyBit\venv\Scripts\python.exe D:/xxx/Issue/PyBit/get_klines.py
<module> Line: 72: result:
{'ext_code': '',
 'ext_info': '',
 'result': [{'close': 56741,
             'high': 56756.5,
             'id': 20953561,
             'interval': '1',
             'low': 56731,
             'open': 56741,
             'open_time': 1638511140,
             'period': '1',
             'start_at': 1638511140,
             'symbol': 'BTCUSDT',
             'turnover': 2094167.757,
             'volume': 36.905},
            {'close': 56823,
             'high': 56824,
             'id': 20953647,
             'interval': '1',
             'low': 56741,
             'open': 56741,
             'open_time': 1638511200,
             'period': '1',
             'start_at': 1638511200,
             'symbol': 'BTCUSDT',
             'turnover': 828316.443,
             'volume': 14.582}],
 'ret_code': 0,
 'ret_msg': 'OK',
 'time_now': '1638511281.109616'}

Process finished with exit code 0
```

As observed, **2094167.757** is the 24 hours turnover rate whereas **36.905** should be the volume.

After the proposed fix:
```
D:\Wecoz\Issue\ccxt\venv\Scripts\python.exe D:/Wecoz/Issue/ccxt/get_klines.py
<module> Line: 71: result:
[[1638511140000, 56741.0, 56756.5, 56731.0, 56741.0, 36.905],
 [1638511200000, 56741.0, 56824.0, 56741.0, 56823.0, 14.582]]

Process finished with exit code 0
```

The OHLCV is now matching with PyBit's result.

**Comment by ttodua** [comment]
@DoctorSlimm  Should we close this issue if it's fixed ?

**Comment by kroitor** [comment]
Looks like Bybit is fixed in #10795. However, this issue also has a report on Bitmart (this exchange is not tradeable at this time, cause they've been hacked a few days ago and now recovering from it).

**Comment by viper7882** [comment]
Thank you @kroitor for the prompt response.
